examples/composed-auth/auth/authorizers.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	errors "github.com/go-openapi/errors"
 	models "github.com/go-swagger/go-swagger/examples/composed-auth/models"
-	jwt "github.com/golang-jwt/jwt"
+	jwt "github.com/golang-jwt/jwt/v5"
```

Comment:
thanks. that's fine. Did you build the examples? Examples are currently not covered by our CI tests.

Reply:
I didn’t built them, just updated the doc.